project/scripted.sbt

```diff
@@ -1,2 +1,2 @@
 //uncomment then will be supported
-libraryDependencies +=  "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
\ No newline at end of file
+// libraryDependencies +=  "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
```

Comment:
Yep. Scipted in sbt 1.0.0-RC is bugged and for now I copy code and fix then problem. See FixScripted.scala